_posts/2018-08-01-toykio.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title:  "Toykio"
+author: "Alexander Polakov"
+author_github: "polachok"
+date:   2018-08-01
+categories: blog
+---
+
+What is toykio?
```

Comment:
Maybe start it like this:

(no headline)

In this blog post I'd like to present how an executor with event loop can be built. What I'm going to present is basically a very simplified version of [Tokio](https://tokio.rs/). I'm calling this project [toykio](https://github.com/polachok/toykio). It provides only minimal feature set: Event loop, ...